docs/content/concepts/assets/external-assets.mdx

```diff
@@ -199,26 +221,50 @@ curl --request POST \
 }'
 ```
 
+---
+
+</TabItem>
+</TabGroup>
+
+#### Using Python
+
+<TabGroup>
+<TabItem name="Dagster Cloud">
+
+##### Dagster Cloud
+
+```python
+import requests
+
+url = f"https://{organization}.dagster.cloud/{deployment}/report_asset_materialization/{asset_key}"
+payload = { "metadata": { "source": "From python script" } }
+headers = { "Content-Type": "application/json", "Dagster-Cloud-Api-Token": "{token}" }
+
+response = requests.request("POST", url, json=payload, headers=headers)
+```
+
+---
+
 </TabItem>
-<TabItem name="Using Python">
+<TabItem name="Dagster OSS">
 
-The following demonstrates how to invoke the API in Python using the `requests` library:
+##### Dagster OSS
 
 ```python
 import requests
 
-url = f"https://path/to/instance/report_asset_materialization/{asset_key}"
+url = f"https://{dagster_webserver_host}/report_asset_materialization/{asset_key}"
 payload = { "metadata": { "source": "From python script" } }
 headers = { "Content-Type": "application/json" }
 
 response = requests.request("POST", url, json=payload, headers=headers)
 ```
 
+---
+
 </TabItem>
 </TabGroup>
 
-The API also has endpoints for reporting [asset observations](/concepts/assets/asset-observations) and [asset check evaluations](/concepts/assets/asset-checks).
```

Comment:
why remove?

Reply:
I just moved it up to come before the examples!